src/deadline/client/cli/_groups/bundle_group.py

```diff
@@ -276,10 +290,42 @@ def bundle_gui_submit(job_bundle_dir, browse, **args):
         response = None
         if submitter:
             response = submitter.create_job_response
-        if response:
+
+        _print_response(
+            output=output,
+            submitted=True if response else False,
+            job_bundle_dir=job_bundle_dir,
+            job_id=response["jobId"] if response else None,
+            parameter_values=submitter.parameter_values,
+            asset_references=submitter.job_attachments.get_asset_references().to_dict(),
+        )
+
+
+def _print_response(
+    output: str,
+    submitted: bool,
+    job_bundle_dir: str,
+    job_id: Optional[str],
+    parameter_values: Optional[list[Dict[str, Any]]],
+    asset_references: Dict[str, Any],
+):
+    if output == "json":
+        if submitted:
+            response = {
+                "status": "SUBMITTED",
+                "jobId": job_id,
+                "jobBundleDirectory": job_bundle_dir,
+                "parameterValues": parameter_values,
```

Comment:
Is it possible to hide some of these extra fields behind additional flags combined with `--output json` e.g. `--output json --output-parameter-values --output-asset-references` or something to that effect. That way only what is actually required is being included and the default behaviour is to give the exact same information as the human readable format?

Reply:
Looks like this was moved behind `--extra-info` so I'll resolve this thread.